gneiss/regression/tests/test_mixedlm.py

```diff
@@ -396,7 +244,62 @@ def test_mixedlm_zero_error(self):
             'real': [1, 2, 3, 4, 5]
         }, index=['s1', 's2', 's3', 's4', 's5'])
         with self.assertRaises(ValueError):
-            mixedlm('real + lame', table, metadata, tree, groups='lame')
+            res = mixedlm('real + lame', table, metadata, tree, groups='lame')
+            res.fit()
+
+    def test_summary(self):
+        np.random.seed(6241)
```

Comment:
This was borrowed directly from the statsmodels test code.  I'm not sure why this was chosen.